api/apitestlib.go

```diff
@@ -363,7 +364,7 @@ func (me *TestHelper) LoginSystemAdmin() {
 }
 
 func GenerateTestEmail() string {
-	if utils.Cfg.EmailSettings.SMTPServer != "dockerhost" {
+	if utils.Cfg.EmailSettings.SMTPServer != "dockerhost" && os.GetEnv("CI_INBUCKET_PORT") == "" {
```

Comment:
> api/apitestlib.go:367:59: undefined: os.GetEnv

You also need this in api4/apitestlib.go.

Reply:
my bad

Reply:
done